### ISSUE TITLE
Why does DiffusionBee connect to amazonaws?

### ISSUE THREAD

**Issue by guser56789** [divamgupta/diffusionbee-stable-diffusion-ui#361]
At launch, DiffusionBee connects to amazonaws.com. Is it for checking app updates or other purposes? Thanks!

**Comment by sblument** [comment]
I believe so, it will notify users when there is an update.

**Comment by AEtheve** [comment]
> I believe so, it will notify users when there is an update.

That's right, amazoaws.com is used to inform customers about the availability of an update and serves just for that purpose.

**Comment by guser56789** [comment]
> > I believe so, it will notify users when there is an update.
> 
> That's right, amazoaws.com is used to inform customers about the availability of an update and serves just for that purpose.

Thanks! Does the server log our IP address, device fingerprint etc.? Just asking out of curiosity.
